tests/test_format_dt.py

```diff
@@ -25,14 +25,23 @@
 import datetime
 import random
 import pytest
-from discord.utils import format_dt
+from discord.utils.public import format_dt, TimestampStyle
 
 # Fix seed so that time tests are reproducible
 random.seed(42)
 
-ALL_STYLES = ["t", "T", "d", "D", "f", "F", "R", None]
+ALL_STYLES: list[TimestampStyle | None] = [
```

Comment:
No need to type a constant since it never changes.

Reply:
Same for all caps constant, dont type them it can actually make the typing less accurate